src/lib/hooks/useTokenList/index.ts

```diff
@@ -14,31 +14,34 @@ import { validateTokens } from './validateTokenList'
 
 export const DEFAULT_TOKEN_LIST = 'https://gateway.ipfs.io/ipns/tokens.uniswap.org'
 
-const chainTokenMapAtom = atom<ChainTokenMap | undefined>(undefined)
+const tokenListAtom = atom<{ list?: string | TokenInfo[]; map?: ChainTokenMap }>({})
```

Comment:
pref change `map` to `tokenMap` and `list` to `tokenInfoList`

Reply:
is `list` the url?